import mysql.connector
from mysql.connector import Error


try:
    conexao = mysql.connector.connect(
        host = "localhost",
        user = "root",
        passwd = "aneis1961",
        database = "casima"
    )
    #criando o cursor
    cursor = conexao.cursor()
    

    cursor.execute("UPDATE animais SET tipo = 'Bovino' WHERE tag = 2")


    #dando commit
    conexao.commit()

    #fechando a conexa
    conexao.close()

    print("Conex達o com o banco feita com sucesso!")
except Error as e:
    aviso = ("N達o foi possivel se conectar ao banco de dados \nErro: " + str(e))
    print("Conex達o com o banco n達o foi sucedida!")